src/components/Icons/Icon.tsx

```diff
@@ -63,27 +79,39 @@ export interface IconProps {
  * Icon component
  */
 export default function Icon(props: React.PropsWithChildren<IconProps>) {
+  const baseClass = "icon";
+
   const {
     blockName,
-    decorative,
+    decorative = false,
     className,
     desc,
     iconRotation,
+    color = IconColors.ui_black,
+    size = IconSizes.large,
     modifiers = [],
     name,
     role,
     title,
     children,
   } = props;
 
-  const baseClass = "icon";
+  const iconModifiers = modifiers;
 
   if (iconRotation) {
-    modifiers.push(iconRotation);
+    iconModifiers.push(iconRotation);
+  }
+
+  if (color) {
+    iconModifiers.push(color);
+  }
+
+  if (size) {
+    iconModifiers.push(size);
   }
 
   const iconProps = {
-    className: bem(baseClass, modifiers, blockName, [className]),
+    className: bem(baseClass, iconModifiers, blockName, [className]),
     role: decorative ? "img" : role,
     "aria-hidden": decorative,
```

Comment:
Would be great to get a bit more detail in the args table about what the _role_ and _title_ props do.  For example, with role we can say something like "Applies an `aria-role` parameter to the icon for accessibility purposes." Would be great to do similar for the _title_ prop.

Reply:
Similar to `desc`, this prop is being used different from how I would think it should be.  I will work on this as well.